Interface/ui_title_text.csv

```diff
@@ -99,60 +99,60 @@ Language_Used#0,"""-"""
-ShipStatus_1#0,"""標準"""
-ShipStatus_2#0,"""混雑"""
-ShipStatus_3#0,"""満員"""
-ShipStatus_4#0,"""休止中"""
-ShipStatus#0,"""混雑状況"""
-CharaMax#0,"""このシップは定員に達したため<br>キャラクターの新規作成を停止しています"""
-TitleEpochLag#0,"""お知らせ"""
+VitaEndTitle#0,"""Noticia para Jogadores no PS Vita"""
+VitaEndText#0,"""Phantasy Star Online 2's support for the PS Vita<br>ended on September 16th, 2020 (Wednesday).<br><br>We would like to express our sincere gratitude<br>to all the players who have used the PS Vita<br>service through its long lifespan.<br><br>To find out how you can continue playing on<br>another platform, please visit the link below.<br>\u3000<&URL pso2.jp_players_news_25893_>"""
+LoginIdSelect_Sega#1,"""Log in no Jogo"""
+LoginIdSelect_UserInfo#1,"""Mudar informacao de Usuario"""
+LoginIdSelect_Credit#1,"""Ver creditos"""
+LoginIdSelect_GameEnd#1,"""Sair do Jogo"""
+LoginIdSelect_LinkSegaId#1,"""Linkar SEGA ID"""
+LoginIdSelect_SegaId_Link#1,"""Migrar para o SEGA ID"""
+LoginVita#1,"""Login e iniciar jogo"""
+LoginCloud#1,"""Log in no jogo"""
+LoginIdSelect_es#1,"""Passo a Passo para linkar sua conta com o PSO2es"""
+LoginIdSelect_LanguageSetting#1,"""Selecione a Lingua usada no Jogo"""
+ShipStatus_0#0,"""Desconhecido"""
+ShipStatus_1#0,"""Normal"""
+ShipStatus_2#0,"""Congestionado"""
+ShipStatus_3#0,"""Cheio"""
+ShipStatus_4#0,"""Offline"""
+ShipStatus#0,"""Status"""
+CharaMax#0,"""Criar personagem nessa Nave foi<br>temporariamente desabilitado, No momento<br>esta na capacidade Maxima."""
+TitleEpochLag#0,"""Anuncio"""
 NotifyEpochLag#0,"""ご利用中の環境の時刻設定と<br>現在の日時に、大幅な差があります。<br><br>ゲーム内の各種機能や表示など<br>一部の動作に影響する恐れがあります。<br><br>時刻を正しくご設定いただいた後、<br>ゲームを再起動されることを<br>おすすめいたします。<br><br>[No.2305]"""
-FormName#0,"""メール認証キーを入力してください"""
-Message#0,"""この環境でプレイするには、「メール認証キー」の<br>入力が必要です。SEGA IDに登録されている連絡先<br>メールアドレスに送信された「メール認証キー」を、<br>以下に入力してください。"""
-Send#0,"""決定"""
-Reload#0,"""メール認証キー再発行"""
-Cancel#0,"""戻る"""
-Error#0,"""メール認証キーを<br>SEGA IDに登録されている<br>メールアドレスに送信しました。"""
-MessageVitaLink#0,"""ご利用中の SEGA ID との連携を<br>開始するには<br><br>メール認証キーの登録が必要です。"""
-UnreceivedSOS#0,"""メールが届かない場合はこちら"""
-FormName#1,"""ワンタイムパスワード認証"""
-Notice#0,"""ワンタイムパスワードを<yellow>半角<c>で入力してください。<br><br>ワンタイムパスワードは発行されてから<br><yellow>約1分間<c>で使えなくなります。<br>その場合は、あらためて「再発行」をお願いします。<br><br>また、<red>6回連続<c>で間違えますと、ロック状態となります。<br>ロック状態の解除に関する情報は<br>プレイヤーズサイトにてご確認ください。"""
-Send#1,"""決定"""
-Cancel#1,"""戻る"""
-WebLinkButton#0,"""認証ができないなどのトラブルはこちら"""
-FormName#2,"""プレイヤーIDネーム入力"""
-Notice#1,"""プレイヤーIDネームとは<br>『ファンタシースターオンライン2』の中で<br>各プレイヤーを判別するための名前のことです。<br><br>設定したプレイヤーIDネームは、<br><yellow>他のユーザーに公開されます。<c><br><br>他のプレイヤーと重複するプレイヤーIDネームは<br>設定できませんのでご注意ください。<br><br>半角文字だけでなく、<yellow>全角文字も使用できます。<c>"""
-Change_FormName#0,"""プレイヤーIDネーム変更"""
+FormName#0,"""Insira Chave para Autenticar"""
+Message#0,"""Voce precisa inserir uma [Chave para Autenticar do E-mail]<br>para jogar. Porfavor insira [Chave para Autenticar do E-mail]<br>que foi enviada para o seu e-mail<br>registrado no seu SEGA ID."""
+Send#0,"""Confirmar""
```

Comment:
There is a quote missing here in file "Interface/ui_title_text.csv", line 125, ID Send#0

Reply:
already fixed